src/components/application_manager/src/commands/hmi/response_from_hmi.cc

```diff
@@ -48,7 +48,7 @@ ResponseFromHMI::ResponseFromHMI(const MessageSharedPtr& message,
   }
 
   // Replace HMI app id with Mobile connection id
-  ReplaceHMIByMobileAppId(*(message.get()));
+  ReplaceHMIWithMobileAppId(*(message.get()));
```

Comment:
Same comment as `request_from_hmi.cc`

Reply:
@jacobkeeler Fixed in 88fc4e0d5a15d60bd2edfc9bd623a54bd71d0243